resources/hooks/local/pre-push

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+remote="$1"
+
+while read local_ref local_sha remote_ref remote_sha
+do
+    # Get the branch name
+    local_ref=$(basename $local_ref)
+    break
+done
+    
+# Fetch the GIT diff and format it as command input:
+DIFF=$(git diff --name-only "$remote"/"$local_ref"..HEAD --oneline)
```

Comment:
Maybe create a new branch of grumphp where work on this feature can be an idea with a roadmap, starting from a refactoring of this pr.